index.ts

```diff
@@ -136,11 +136,15 @@ export {CurrencyFormat} from './schema';
 export type {CurrencySchema} from './schema';
 export type {DurationSchema} from './schema';
 export {DurationUnit} from './schema';
+export type {EmailDisplayType} from './schema';
```

Comment:
Nit, I think you want to do a regular export on these two enums, so their values can be referenced too.